ios/NativeModules/TextileIPFS.m

```diff
@@ -259,7 +259,7 @@ - (NSString *)_getHashRequest:(NSString *)hash error:(NSError**)error {
   return token;
 }
 
-- (NSString *)_getPhoto:(NSString *)hashPath error:(NSError**)error {
+- (NSString *)_getHashData:(NSString *)hashPath error:(NSError**)error {
```

Comment:
as with all the others, i think this is a good move: renaming to be more generic